ios/Sources/SettingFeature/SettingScreen.swift

```diff
@@ -1,69 +1,48 @@
 import Component
+import ComposableArchitecture
 import Introspect
 import Styleguide
 import SwiftUI
 
-public enum SettingModel: Hashable {
-    case darkMode(isOn: Bool)
-    case language(isOn: Bool)
-
-    var title: String {
-        switch self {
-        case .darkMode:
-            return L10n.SettingScreen.ListItem.darkMode
-        case .language:
-            return L10n.SettingScreen.ListItem.language
-        }
-    }
-
-    var isOn: Bool {
-        switch self {
-        case let .darkMode(isOn), let .language(isOn):
-            return isOn
-        }
-    }
-
-    mutating func update(isOn: Bool) {
-        switch self {
-        case .darkMode:
-            self = .darkMode(isOn: isOn)
-        case .language:
-            self = .language(isOn: isOn)
-        }
-    }
-}
-
 public struct SettingScreen: View {
 
-    @State private var items: [SettingModel]
-
     @Environment(\.presentationMode) var presentationMode
+    private let store: Store<SettingState, SettingAction>
 
-    public init(isDarkModeOn: Bool, isLanguageOn: Bool) {
+    public init(
+        isDarkModeOn: Bool, isLanguageOn: Bool
+    ) {
         let darkModeModel = SettingModel.darkMode(isOn: isDarkModeOn)
         let languageModel = SettingModel.language(isOn: isLanguageOn)
-        _items = State(initialValue: [darkModeModel, languageModel])
+        self.store = .init(
+            initialState: .init(items: [darkModeModel, languageModel]),
+            reducer: settingReducer,
+            environment: SettingEnvironment()
+        )
```

Comment:
Is it possible to give store from AppFeature, not in `init()`? I think it's better with AppFeature.

Reply:
I got it. 👍